plugins/rapid7_insightvm/komand_rapid7_insightvm/util/update_shared_credential_util.py

```diff
@@ -0,0 +1,207 @@
+from typing import Union
+
+from insightconnect_plugin_runtime.exceptions import PluginException
+
+from komand_rapid7_insightvm.util.util import check_in_enum, check_not_null
+
+
+def ssh_setup(account: dict) -> tuple[str, str, str]:
+    permission_elevation = account.get("permission_elevation", "none")
+    check_in_enum(
+        permission_elevation, "permission_elevation", ["none", "sudo", "sudosu", "su", "pbrun", "privileged-exec"]
+    )
+    if permission_elevation not in ("none", "pbrun"):
+        permission_elevation_username = check_not_null(account, "permission_elevation_username")
+        permission_elevation_password = check_not_null(account, "permission_elevation_password")
+    else:
+        permission_elevation_username = ""
+        permission_elevation_password = ""  # nosec
+    return permission_elevation, permission_elevation_username, permission_elevation_password
+
+
+def usr_and_pass(account: dict):
```

Comment:
Kindly please the return typehint and change dict to `Dict` from typing with a proper structure types